Model/DbConnection.php

```diff
@@ -0,0 +1,445 @@
+<?php
+
+  // Loaded all required libraries.
+  require("../vendor/autoload.php");
+
+  // Loading .env credentials.
+  $dotEnv = Dotenv\Dotenv::createImmutable(__DIR__);
+  $dotEnv->safeLoad();
+
+  /**
+   * Connects with database using mysqli, and provide more functionality to for database.
+   * 
+   *  @method existsUserId
+   *    Checks if user's user_id exists.
+   * 
+   *  @method existsMailId
+   *    Checks if user's mail id exists. 
+   * 
+   *  @method getQueryArray
+   *    Sends query response as array
+   * 
+   *  @method getPhotoURLbyId
+   *    Get photo url of user by id. 
+   * 
+   *  @method getUsersCount
+   *    Get counts of all users. 
+   * 
+   *  @method getPhotoURLbyUserId
+   *    Get photo url of user by user id. 
+   * 
+   *  @method getPass
+   *    Get Password of given userId.
+   * 
+   *  @method getName
+   *    Get name of given userId.
+   * 
+   *  @method getNamebyId
+   *    Get name of given ID. 
+   * 
+   *  @method getId
+   *    Get Id of given userId. 
+   * 
+   *  @method getAbout
+   *    Get About of given userId. 
+   * 
+   *  @method getAllUsers
+   *    Get all users of ordered by given order, by default it order by first_name.
+   * 
+   *  @method getAllPosts
+   *    Get limit number of posts of ordered by given order.
+   * 
+   *  @method getOnline
+   *    Get online status of given userId. 
+   * 
+   *  @method getLikeCountbyPostId
+   *    Get like count of given postId. 
+   * 
+   *  @method getLikedPostId
+   *    Get postIds of liked posts only.
+   * 
+   *  @method getPostIdbyUserId
+   *    Get post id of given userId as string. 
+   * 
+   *  @method setPost
+   *    Set new created post to database. 
+   * 
+   *  @method setUser
+   *    Sets users details. 
+   * 
+   *  @method setOnline
+   *    Update online status of given user id.
+   * 
+   *  @method deletePost
+   *    Delete post of given postId. 
+   * 
+   *  @var mysqli $conn
+   *    Update online status of given user id. 
+   *
+   * */
+  class DbConnection {
+    // Define global $conn to hold connection object after successful authentication with database.
+    public mysqli $conn;
```

Comment:
Proper Comment for  variable missing. Use multiline comment for class variables.